.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Are you filing an issue about iOS 12?**
+Please see preliminary support for iOS 12 from PR: https://github.com/openid/AppAuth-iOS/pull/246 before filing iOS 12 issues. Thanks.
```

Comment:
I'd use https://github.com/openid/AppAuth-iOS/tree/ios12beta as the link, people can point their preferred dependency manager at that.

Reply:
Done. PTAL?